portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,84 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css" />
+        <link rel="stylesheet" href="style.css" />
+        <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+        <script src="node_modules/jquery/dist/jquery.min.js"></script>
+        <script src="script.js"></script>
```

Comment:
I think you'll want to delete script.js and the reference here since it's empty in this PR.